### PR TITLE
Add the -e parameter and call wrapper

### PR DESCRIPTION
**Pull request by Stevent-fei**

### CODE REVIEW

File: sealer/cmd/run.go [hunk 1]
```diff
@@ -68,6 +68,7 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '10.233.0.0/18'")
 	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "", "", "set default service CIDR network. example '10.233.64.0/18'")
+	runCmd.Flags().StringVarP(&runArgs.CustomEnv, "customEnv", "e", "", "set custom environment variables. example 'sealer run -e DashBoardPort=8443 mydashboard:latest'")
```
Comment (bxy4543) [inline sealer/cmd/run.go:71]: string type is replaced with the slice type（runCmd.Flags().StringSliceVarP）

runArgs.CustomEnv needs to be written to the cluster in `NewApplierFromArgs(args[0], runArgs)` function

File: pkg/plugin/shell_plugin.go [hunk 2]
```diff
@@ -80,3 +83,20 @@ func (s Sheller) Run(context Context, phase Phase) error {
 func init() {
 	Register(ShellPlugin, &Sheller{})
 }
+
+func CustomEnv(context Context) (env.Interface, error) {
```
Comment (fanux) [inline pkg/plugin/shell_plugin.go:87]: Where do you call this function, and why return the env.Interface.

File: common/structs.go [hunk 3]
```diff
@@ -24,4 +24,5 @@ type RunArgs struct {
 	PodCidr    string
 	SvcCidr    string
 	Provider   string
+	CustomEnv  []string
```
Comment (kakaZhou719) [inline common/structs.go:27]: i think the arg `CustomEnv ` should be `map[string]string`  type. why we use `[]string`.

Comment (kakaZhou719) [inline common/structs.go:27]: I’m curious about why this structure `RunArgs` defined in common? it should in apply module.

Comment (Stevent-fei) [reply common/structs.go:27]: My understanding is that the execution command, for example, sealer run image -e port=6644 -m xxx -n xxx -p xxx should be in the run parameter, and the run parameter is in the common module.

**Review by fanux — APPROVED** [review]

**Pull request merged**